draft-westerlund-tsvwg-sctp-crypto-chunk.md

```diff
@@ -713,94 +714,21 @@ an ABORT chunk SHALL NOT be sent. This way non-critical errors
 are handled and how the protection engine will recover from
 these errors is being described in the Protection Engine Specifications.
 
-# Protected SCTP State Diagram {#state-diagram}
+## Integration with SCTP State Machine {#new-states}
```

Comment:
Maybe better to call it state diagram. The term "state machine" is not used in 9260